api/propertes_windows.go

```diff
@@ -0,0 +1,18 @@
+package api
```

Comment:
typo in file name `propertes` -> `properties`

Reply:
+1, please fix the typo in the filename here.